lib/src/model/game/game_history.dart

```diff
@@ -33,15 +33,19 @@ const _nbPerPage = 20;
 /// stored locally are fetched instead.
 @riverpod
 Future<IList<LightArchivedGameWithPov>> myRecentGames(
-  MyRecentGamesRef ref,
-) async {
+  MyRecentGamesRef ref, {
+  GameFilterState filter = const GameFilterState(),
```

Comment:
Recent games should not be filtered (whether it is my recent games or any user recent games) so this parameter should be removed.

Reply:
The game history provider uses the recent games providers in its implementation. We could create new providers that do the same thing, except they watch the game filter provider instead of passing it as a param. I don't really see the point though.

Reply:
I'd rather make the request again than have an extra complexity and unnecessary filter in the recent game providers.

So you can just remove the dependency to the recent game provider in `UserGameHistory` and perform the request directly with the filters.